src/sync-agent/namespace.ts

```diff
@@ -103,6 +106,8 @@ function createDefaultNamespace(): SyncAgentNamespace {
     syncKeyword: KeywordComponent.create('sync'),
     atLeastOnceKeyword: KeywordComponent.create('alo'),
     latestOnlyKeyword: KeywordComponent.create('late'),
+    // Adam Chen Add snapshotKeyword
+    snapshotKeyword: KeywordComponent.create('snapshot'),
```

Comment:
This is for agnostic namespace usage, i.e. creating or parsing a name without knowledge of the namespace design. Do you put it here for document only? If so, you can leave a comment. If not, you can add a function to use this constant.

Reply:
I thought this is for storing the identifiers used by serve().
I will move them out in a separate commit, so that its easier for us to move part 2 out into its separate function later down the line.